util/size_regression.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -eEuo pipefail
+
+job_count=$(getconf _NPROCESSORS_ONLN 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null || echo 2)
+source_ref="0.11.0"
+dest_ref="develop"
+ignore_ref="master"
+
+function usage() {
+    echo "Usage: $(basename "$0") [-s <source>] [-d <dest>] [-u] planck/rev6:default"
```

Comment:
Complains about
```
❯ ./util/size_regression.sh -u planck/rev6:default                                                                                                                                                                                                                                                              
./util/size_regression.sh: illegal option -- u
```

Probably a leftover?

Reply:
Yep. Updated to reflect the current list of args.